group_vars/all/k8s

```diff
@@ -1,4 +1,3 @@
 ---
 # Cloud provider to use for Kubernetes clusters
 cloud_provider: openstack
-...
```

Comment:
outrageous

Reply:
I have decided to abandon the practice. Its a lot of pain for little gain.